### DISCUSSION TITLE
Acteur reflet d'une variable analogique

### DISCUSSION THREAD

**Discussion by JYK-WIT** [witsa/synapps#524] [Q&A] [answered]
Bonjour,
lors de l'affichage d'un reflet analogique avec forçage, les seuils apparaissent alors qu'ils ne sont pas demandés.
De plus si on renseigne des limites, un slider apparait pour la sélection de la valeur , mais ne disparait pas après validation.
Peut on avoir le choix de visualisation.
Merci
JYK

**Comment by manufitoussiwit** [comment, marked as answer]
Salut!

Malheureusement, l'acteur *détails de reflet* ne permet pas de modifier l'aspect qui est spécifique au type de reflet choisi. 
En revanche, il y a un moyen de caché ce que tu ne désire pas voir. En ajoutant dans la même scène que l'acteur détails l'acteur *html* suivant : 

```text
SYNAPPS-STUDIO-ACTOR|{"type":"display/html","key":"css-analog-custom","properties":{"content":"<style>\n  .reflect-details-analog .input-cursor,\n  .reflect-details-analog .input-cursor + div {\n    display: none !important;\n  }\n\n  .reflect-details-analog button.details-content-item {\n    display: none !important;\n  } \n</style>"}}
```

> Attention! L'acteur va agir sur tout le acteurs details qui sont actuellement affichés

> Attention n°2! Il va agir sur tous les reflets analogiques mais on peux faire en sorte de l'activer uniquement sur le bon.
